app/views/admin/terms_of_service/drafts/show.html.haml

```diff
@@ -14,6 +14,9 @@
   .fields-group
     = form.input :changelog, wrapper: :with_block_label, input_html: { rows: 8 }
 
+  .fields-group
+    = form.input :effective_date, wrapper: :with_block_label, as: :date
```

Comment:
Should we use a native HTML5 date field?

Reply:
I gave that a try (just need to add `html5: true` as an option to the line) but the pop-up widget looks kind of weird on Firefox/Linux and Firefox/Windows so I would prefer not to; the other date inputs we have don't use it either.

Reply:
We have other date inputs that don't use it? The only ones I can think of are those from the announcement interface, and they definitely use it…